.github/workflows/test-shared.yml

```diff
@@ -95,6 +95,11 @@ jobs:
         with:
           extra_nix_config: sandbox = true
 
+      - uses: cachix/cachix-action@0fc020193b5a1fa3ac4575aa3a7d3aa6a35435ad # v16
+        with:
+          name: nodejs
+          authToken: ${{ secrets.CACHIX_AUTH_TOKEN }}
```

Comment:
What's that?

Reply:
I've opened https://github.com/nodejs/node/pull/60428 to discuss that

Reply:
I doesn't explain where the secret comes from / how it's managed/rotated.